assets/javascripts/src/actions.es6

```diff
@@ -28,6 +32,47 @@ export const GA_ENABLED = "GA_ENABLED";
 
 export const AUTOFILL = "AUTOFILL";
 
+export function openStripeCheckout(dispatch) {
+    const state = store.getState();
+    console.log('is this thing on');
+    let hide = () => {
+        dispatch({type:CLOSE_STRIPE});
+    };
+    let process = (token) => {
+        dispatch({ type: SUBMIT_PAYMENT });
+        let data = paymentFormData(state, token.id);
+        fetch(urls.pay, {
+            credentials: 'same-origin',
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify(data)
+        })
+        .then(response => response.json() // response: Response
+            .then(json => {
+                return { response: response, json: json }
+            }))
+            .then((response) => {
+                if (response.response.ok)
```

Comment:
it does get a little confusing but i think the idea is to persist the original `response` object along with the parsed json... it might be worth writing a `fetch-json` module that does this